scripts/linters/pylint_extensions.py

```diff
@@ -2109,6 +2109,46 @@ def visit_call(self, node):
                     break
 
 
+class DisallowBlankLinesBelowFunctionDefinitionChecker(checkers.BaseChecker):
+    """Custom pylint checker to ensure there are no blank lines
+    below function definition.
+    """
+
+    __implements__ = interfaces.IAstroidChecker
+    name = 'disallow-blank-lines-below-function-definition'
+    priority = -1
+    msgs = {
+        'C0034': (
+            'Please remove the blank line(s) below the function definition.',
+            'remove-blank-lines-below-function-definiton.',
+            'Disallow blank lines below function definition.'
+        ),
+    }
+
+    def visit_functiondef(self, node):
+        """Visits every function definition and ensures there are no blank lines
+        below function definition.
+
+        Args:
+            node: astroid.scoped_nodes.Function. Node for a function or method
+                definition in the AST.
+        """
+        c = 0
+        for i in python_utils.RANGE(len(node.root().file)):
+            c += 1
```

Comment:
PLease use a better variable name, I'm not sure what's going on here?

Reply:
Corrected. I have used proper variable names now.